docs/README.md

```diff
@@ -1,2 +1,204 @@
+
+##### court
+
+The `court` object represents the court details.
+
+##### Properties
+
+| name       | type   | description                                                                   |
+| ---------- | ------ | ----------------------------------------------------------------------------- |
+| location   | string | Court House where the package is submitted to                                 |
+| level      | string | Court level, can be `Supreme` of `Provincial`                                 |
+| courtClass | string | Court Classification see [Court Classification](data.md#Court-Classification) |
+| division   | string | R for Criminal, I for Civil see [Court Division](data.md#Court-Division)      |
+| fileNumber | string | The court file number                                                         |
+
+##### Documents
+
+The `documents` array represents the previously uploaded document.
+
+###### Properties
+
+| name                     | type    | description                                                                                |
+| ------------------------ | ------- | ------------------------------------------------------------------------------------------ |
+| name                     | string  | the document name, must be the same name as uploaded previously                            |
+| type                     | string  | the type of document see [Document Type Codes](data.md#Document-Type-Codes)                |
+| isAmendment              | boolean | if the document is an amendment                                                            |
+| isSupremeCourtScheduling | boolean | is the document is directed to Supreme Court Scheduling                                    |
+| data                     | object  | A non define json object representing the form data of the document                        |
+| md5                      | string  | The md5 hash value of the document content, used to validate the integrity of the document |
+
+##### parties
+
+The `parties` array represents a list of parties for the submission
+
+##### Properties
+
+| name       | type   | description                                                                                     |
+| ---------- | ------ | ----------------------------------------------------------------------------------------------- |
+| partyType  | string | the party type, IND for individual or ORG for Organization see [Party Type](data.md#Party Type) |
+| roleType   | string | the party role, see [Party Role](data.md#Party-Role)                                            |
+| firstName  | string | the party first name                                                                            |
+| middleName | string | the party middle name                                                                           |
+| lastName   | string | the party last name                                                                             |
+
```

Comment:
as part of the generate url payload info we should also include a section about the validation.

1) must provide a court file number. If this is not provided, then parties with valid role type are mandatory

2) must provide valid document types

3) must provide a valid court level, class, and location combination

Reply:
I have added a required column in all the tables